### PR TITLE
Patch relnotes.k8s.io to release v1.28.0-alpha.3

### PR DESCRIPTION
**Pull request by rashansmith**
Automated patch to update relnotes.k8s.io to k/k version `v1.28.0-alpha.3`

### CODE REVIEW

**Comment by saschagrunert** [comment]
Hm, we probably need more resources for this test

/retest

**Review by saschagrunert — APPROVED** [review]

File: src/assets/release-notes-1.28.0-alpha.3.json [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/115835",
+    "pr_number": 115835,
+    "areas": ["test", "kubelet"],
+    "kinds": ["bug"],
+    "sigs": ["node", "testing"],
+    "duplicate": true
+  },
+  "116530": {
+    "commit": "03486cec21579ce77304bb7265ecb90516418301",
+    "text": "kubelet: un-deprecate --provider-id flag",
```
Comment (sanchita-07) [inline src/assets/release-notes-1.28.0-alpha.3.json:41]: `--provider-id`

File: src/assets/release-notes-1.28.0-alpha.3.json [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+    "pr_number": 117702,
+    "areas": ["test", "kubelet"],
+    "kinds": ["feature"],
+    "sigs": ["node", "testing"],
+    "feature": true,
+    "duplicate": true
+  },
+  "117793": {
+    "commit": "86d786090ac9390d4b724a7294e8bb96825eeefe",
+    "text": "If using cgroups v2, then the cgroup aware OOM killer will be enabled for container cgroups via  `memory.oom.group` .  This causes processes within the cgroup to be treated as a unit and killed simultaneously in the event of an OOM kill on any process in the cgroup.",
```
Comment (sanchita-07) [inline src/assets/release-notes-1.28.0-alpha.3.json:128]: I guess there are some spacing issues that we can remove.

File: src/assets/release-notes-1.28.0-alpha.3.json [hunk 3]
```diff
@@ -0,0 +1,364 @@
...
+    "pr_number": 118128,
+    "areas": ["test", "kubelet", "kube-proxy", "code-generation"],
+    "kinds": ["cleanup", "api-change", "deprecation"],
+    "sigs": ["network", "storage", "node", "api-machinery", "apps", "testing"],
+    "duplicate": true,
+    "duplicate_kind": true
+  },
+  "118278": {
+    "commit": "1de217b095988a0e76b2ac4f3ebcfcbf4bb5f08e",
+    "text": "Update the comment about the feature-gate level for PodFailurePolicy from alpha to beta",
```
Comment (sanchita-07) [inline src/assets/release-notes-1.28.0-alpha.3.json:182]: Update -> Updated
Backticks in PodFailurePolicy

File: src/assets/release-notes-1.28.0-alpha.3.json [hunk 4]
```diff
@@ -0,0 +1,364 @@
...
+    "author_url": "https://github.com/mimowo",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/118278",
+    "pr_number": 118278,
+    "kinds": ["bug", "documentation", "api-change"],
+    "sigs": ["apps"],
+    "duplicate_kind": true
+  },
+  "118329": {
+    "commit": "9d231f8ce77b4a8413b41130ea130d392ca860c7",
+    "text": "Fixes the alpha `CloudDualStackNodeIPs` feature.",
```
Comment (sanchita-07) [inline src/assets/release-notes-1.28.0-alpha.3.json:194]: Fixes->Fixed

File: src/assets/release-notes-1.28.0-alpha.3.json [hunk 5]
```diff
@@ -0,0 +1,364 @@
...
+    "areas": ["kubelet"],
+    "kinds": ["bug", "feature"],
+    "sigs": ["network", "node"],
+    "feature": true,
+    "duplicate": true,
+    "duplicate_kind": true
+  },
+  "118420": {
+    "commit": "b63700630231803967f85e83ff35d719db511baa",
+    "text": "ACTION_REQUIRED\nWhen an Indexed Job has a number of completions higher than 10^5 and parallelism higher than 10^4, and a big number of Indexes fail, Kubernetes might not be able to track the termination of the Job. Kubernetes now emits a warning, at Job creation, when the Job manifest exceeds both of these limits.",
```
Comment (sanchita-07) [inline src/assets/release-notes-1.28.0-alpha.3.json:216]: I think we can remove the ACTION_REQUIRED, as they are not a part of the release note.

File: src/assets/release-notes-1.28.0-alpha.3.json [hunk 6]
```diff
@@ -0,0 +1,364 @@
...
+    "author_url": "https://github.com/serathius",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/118460",
+    "pr_number": 118460,
+    "areas": ["apiserver"],
+    "kinds": ["bug"],
+    "sigs": ["api-machinery"]
+  },
+  "118530": {
+    "commit": "bca13360215bbd2aad562207dcf0dd8a6a31779f",
+    "text": "Fix Cronjob status.lastSuccessfulTime not populated by a manually triggered job",
```
Comment (sanchita-07) [inline src/assets/release-notes-1.28.0-alpha.3.json:247]: Fixed Cronjob `status.lastSuccessfulTime`

File: src/assets/release-notes-1.28.0-alpha.3.json [hunk 7]
```diff
@@ -0,0 +1,364 @@
...
+    "author_url": "https://github.com/carlory",
+    "pr_url": "https://github.com/kubernetes/kubernetes/pull/118530",
+    "pr_number": 118530,
+    "areas": ["kubectl"],
+    "kinds": ["bug"],
+    "sigs": ["cli"]
+  },
+  "118552": {
+    "commit": "471fd1ae8a1d167102f1d6d27fbde1020d320c59",
+    "text": "RBD volume plugin ( `kubernetes.io/rbd`) has been deprecated in this release and will be removed in a subsequent release. Alternative is to use RBD CSI driver (https://github.com/ceph/ceph-csi/) in your Kubernetes Cluster.",
```
Comment (sanchita-07) [inline src/assets/release-notes-1.28.0-alpha.3.json:259]: There is an extra space.

**Review by sanchita-07 — COMMENTED** [review]
Looks good. Just a few suggestions.

**Review by sanchita-07 — APPROVED** [review]
/lgtm

**Pull request merged**